jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -257,4 +400,52 @@ def _on_viewer_key_event(self, viewer, data):
             x, y = row_info['axes_x'], row_info['axes_y']
             self._get_mark(viewer).append_xy(getattr(x, 'value', x), getattr(y, 'value', y))
 
-            self.hub.broadcast(MarkersPluginUpdate(table_length=len(self.table), sender=self))
+        # on mac 'option + d' gets transmitted as delta instead of just d + alt
+        elif data['event'] == 'keydown' and data.get('key') in ('d', '∂'):
+            if data.get('altKey', False):
+                coords = self._get_snap_coordinates(viewer)
+            else:
+                coords = self.coords_info.as_dict()
+
+            viewer_id = viewer.reference or viewer.reference_id
+
+            if self._distance_first_point is None:
+                self._distance_first_point = coords
+                self.distance_display = "..."
+            else:
+                p1 = self._distance_first_point
+                p2 = coords
+                self._distance_first_point = None
+
+                if self.app._align_by.lower() == 'wcs':

```

Comment:
I just realized that this only works for Imviz. In Cubeviz, the distance label shows as pixel even though there are WCS coordinates available, since (I assume) this `align_by` check fails. The distance in arcsec is still available in the table, but we should probably display the world distance as the label on the line as well.